File: java/home_work_1/task13.java
import java.util.*;
class Task1 {
	static int multiplier(int x, int y, int z) {
	  return x * y *z;
	}
    public static void main(String[] args) {
        
	int x,y,z, mul;	
	System.out.println("Please Enter X");
	System.out.println("Please Enter Y");
	System.out.println("Please Enter Z");
        Scanner sx = new Scanner(System.in);
        Scanner sy = new Scanner(System.in);
        Scanner sz = new Scanner(System.in);
	x = sx.nextInt();	
	y = sy.nextInt();	
	z = sz.nextInt();	
	mul = multiplier(x,y,z);
	System.out.println("x * y * z = " + mul);
    }
}

File: cpp/home_work_1/task13.cpp
#include <iostream>
    int multiplier(int x, int y, int z){
	return x*y*z;
    }
int main() {
    int x,y,z, mul;
    std::cout << "Please Enter X, Y, Z" << std::endl;
    std::cin >> x >> y >> z;
   
    mul = multiplier(x,y,z);

    std::cout << "Result is " << mul << std::endl;

    return 0;
}
